### PR TITLE
More coding style checks

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: scripts/check_style.sh [hunk 1]
```diff
@@ -17,14 +17,16 @@ fi
 
 FORMATERROR=$(
 (
-git grep -nIE "\<(if|for)\(" -- '*.h' '*.cpp'
-git grep -nIE "\<if\>\s*\(.*\)\s*\{\s*$" -- '*.h' '*.cpp'
+	git grep -nIE "\<(if|for)\(" -- '*.h' '*.cpp' # no space after "if" or "for"
+	git grep -nIE "\<if\>\s*\(.*\)\s*\{\s*$" -- '*.h' '*.cpp' # "{\n" on same line as "if" / "for"
+	git grep -nIE "\(const " -- '*.h' '*.cpp' # const on left side of type
```
Comment (ekpyron) [inline scripts/check_style.sh:22]: Thanks - I really need this check :-D.
And:
``[a-zA-Z0-9_] *&[a-zA-Z]`` or something like that maybe?

**Review by leonardoalt — APPROVED** [review]

File: scripts/check_style.sh (hunk 1 above)
Comment (chriseth) [reply scripts/check_style.sh:22]: That's something you can add :)

Comment (chriseth) [reply scripts/check_style.sh:22]: Actually we could also add `,const`

Comment (ekpyron) [reply scripts/check_style.sh:22]: Ok, I'll add both :-).

**Pull request merged**